package.json

```diff
@@ -248,11 +249,32 @@
             "off"
           ],
           "enumDescriptions": [
-            "Refresh memory views when when debugger stops (e.g. a breakpoint is hit)",
-            "Memory view data is manually refreshed by user"
+            "Refresh Memory Inspector when the debugger stops (e.g. a breakpoint is hit)",
+            "Do not automatically refresh when the debugger stops"
           ],
           "default": "on",
-          "description": "Refresh memory views when debugger stops"
+          "description": "Refresh Memory Inspectors when the debugger stops"
+        },
+        "memory-inspector.periodicRefresh": {
+          "type": "string",
+          "enum": [
+            "always",
+            "while running",
+            "off"
+          ],
+          "markdownEnumDescriptions": [
+            "Always refresh automatically after the configured `#memory-inspector.periodicRefreshInterval#`",
+            "Refresh automatically after the configured `#memory-inspector.periodicRefreshInterval#` while the CPU is running",
+            "Do not automatically refresh after the configured delay"
+          ],
+          "default": "off",
+          "description": "Refresh Memory Inspectors after the configured delay"
```

Comment:
Let's include the link to the other preference here as well. That'll be easier for users to click on and make the connection more obvious.

Reply:
Good idea, I'll add the link!